lua/neogit/buffers/commit_view/init.lua

```diff
@@ -281,7 +281,7 @@ function M:open(kind)
         end),
         [popups.mapping_for("RemotePopup")] = popups.open("remote"),
         [popups.mapping_for("RevertPopup")] = popups.open("revert", function(p)
-          p { commits = { self.commit_info.oid } }
+          p { commits = { self.commit_info.oid }, hunk = self.buffer.ui:get_hunk_or_filename_under_cursor() }
```

Comment:
```suggestion
          p { commits = { self.commit_info.oid }, hunk = self.buffer.ui:get_hunk_or_filename_under_cursor().hunk }
```

I think it's 2% nicer to pass in the hunk data directly, instead of having two levels of `hunk` keys. Requires updating the consumer below in revert actions.

Reply:
`get_hunk_or_filename_under_cursor` can return `nil` so cannot the `hunk` key until I check if `nil`. I renamed this key to item so it reads a bit better.

Reply:
Arg, right. Ok, no reason to overhaul the API then, it's fine as is :)